pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Managing schemas using Schema Registry
+description: The guide explains how you can manage schemas using Schema Registry.
+weight: 50
+---
+
+## Introduction
+This tutorial will teach you how to manage schemas with Schema Registry. This will enable us to ensure a consistent exchange of data across distributed systems, thus minimizing any errors.
+
+## Background context
+To handle the evolving complexity of modern Event-Driven Architecture, the need for schema management has become increasingly prevalent. Data validation is a technique that helps in maintaining data consistency and compatibility. Schema Registry can be used to validate data being exchanged by producers and consumers. 
+
+In this tutorial, we’ve used the Apicurio Registry, which is a popular Schema Registry implementation, in combination with Avro which is a language-neutral data serialization system.
+
+## Prerequisites
+Before you begin with this tutorial, it is already assumed that you already have:
+- Set up an instance of Apicurio Registry 
```

Comment:
It's good we have this but to me it should be a pointer to Apicurio docs. Otherwise, we'll have to maintain this document if Apicurio changes.

Reply:
cc @derberg for this question too ⬆️